test/000_fullchain.test.ts

```diff
@@ -299,6 +300,135 @@ describe('Integration tests', function () {
                 for (let i = 0; i < workerNumber; i++) {
                     if (workerNumber == 1) {
                         expect(await iexecPoco.viewScore(workers[i].address)).to.be.equal(0);
+                    } else {
```

Comment:
Maybe `expect(await iexecPoco.viewScore(workers[i].address)).to.be.equal(workerNumber == 1? 0 : 1);
`

Reply:
updated [here](https://github.com/iExecBlockchainComputing/PoCo/pull/160/commits/5c6045a181b7d5a60e17355026900ac504cfffaa)